common/culture/traditions/00_regional_traditions.txt

```diff
@@ -426,7 +442,14 @@ tradition_caravaneers = {
 	}
 	
 	is_shown = {
-		always = no
+		# Warcraft
+		any_culture_county = {
+			any_county_province = {
+				geographical_region = custom_deserts
+				geographical_region = custom_drylands
+				geographical_region = custom_oasis
```

Comment:
Not sure it's good idea. May look better if it's culture locked to vulpera, pygmies, etc. Kezan is a dryland too.